uberon_edit.obo

```diff
@@ -193632,15 +193642,6 @@ intersection_of: UBERON:0006983 ! anatomical point
 intersection_of: distalmost_part_of UBERON:0002415 ! tail
 created_by: https://orcid.org/0000-0003-2699-3066
 
-[Term]
-id: UBERON:7500036
```

Comment:
Hi, it looks like you deleted this term. Was this deliberate? It looks like you added this in the first place so I will leave it to you whether or not you want to keep it - however, we have a policy of no deletions, IDs must be retained. You can use the obsolete entity (GO workflow) option in Protege to obsolete this.

Reply:
@cmungall did my changes come through?